capstone-app/src/app/modules/photos/components/image-container/image-container.component.scss

```diff
@@ -1,19 +1,18 @@
-  .flex-container {
-    display: flex;
-    flex-wrap: wrap;
-    font-family: Georgia;
-    letter-spacing: 1.5px;
-    margin-left: 43px;
-    margin-right: 43px;
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Roboto";
+  font-weight: 500;
+  letter-spacing: 1.5px;
 
-    .picture {
-      border-radius: 3px;
-      box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.05);
-      display: inline-block;
-      opacity: 0.99;
-      overflow: hidden;
-      position: relative;
-      text-align: left;
+  .picture {
+    border-radius: 3px;
+    box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.05);
+    display: inline-block;
+    opacity: 0.99;
+    overflow: hidden;
+    position: relative;
```

Comment:
I'd recommend against relative positioning. Is there an alternative?

Reply:
The relative positioning is somewhat crucial to how I've sent up the responsive image grid (my first PR). Is this a must change?